code/datums/movement_controller/pod.dm

```diff
@@ -114,10 +114,12 @@
 
 				//normalize and force speed cap
 				velocity_magnitude = vector_magnitude(velocity_x, velocity_y)
-				var/vel_max = velocity_max + max(owner.speed,0)
+				var/vel_max = velocity_max
 				if (!input_x && !input_y)
 					vel_max = velocity_max_no_input
 
+				vel_max /= (owner.speed ? owner.speed : 1)
```

Comment:
What exactly is the difference here? They don't seem to be equivalent. was the original one wrong? Yours uses it as a divisor, but the original just added it.

Reply:
Changed it to a divisor as part of tweaking the speeds of pods. Maybe a multiplier would be more intuitive...